src/layouts/Header.tsx

```diff
@@ -196,9 +208,11 @@ class Header extends React.Component<IProps, null> {
                 icons={{ checked: <img src='/assets/images/Icon/checked.svg'/>, unchecked: <img src='/assets/images/Icon/unchecked.svg'/> }}/>
             </div>
           </TrainingTooltip>
-          <div className={css.redemptionsButton}>
-            <RedemptionsButton currentAccountAddress={currentAccountAddress} />
-          </div>
+          {
+            this.props.showRedemptionsButton ? <div className={css.redemptionsButton}>
+              <RedemptionsButton currentAccountAddress={currentAccountAddress} />
+            </div> : <div></div>
```

Comment:
should not have an empty div here, can screw up the layout

Reply:
fixed with: https://github.com/daostack/alchemy/pull/1420/commits/6e7d4665d341999f477a1009d51ed9dbe48169c7